src/components/Tooltip.test.tsx

```diff
@@ -26,4 +27,9 @@ describe("Tooltip", () => {
     expect(resolveTooltip(undefined, "Test 2", true)).toBe("Test 2");
     expect(resolveTooltip(undefined, "Test 2", "")).toBe("Test 2");
   });
+
+  it("renders with title prop", async () => {
+    const r = await render(<Tooltip title="Test tooltip">Text</Tooltip>);
+    expect(r.tooltip()).toHaveAttribute("title", "Test tooltip");
```

Comment:
Should we do `data-title` or is `title` a valid attribute for spans?

Reply:
`title` is a valid attribute.